pkg/policy/address.go

```diff
@@ -128,8 +142,8 @@ func ValidateDomainPart(domain string) bool {
 			hasAlphaNum = true
 			labelLen++
 		case c == '-':
-			if prev == '.' {
-				// Cannot lead with hyphen.
+			if prev == '.' || prev == '-' {
```

Comment:
I do have a question regarding L131...
Is this purposefully meant to allow domains ending with a dot?
I originally had a test that would verify that this would be rejected, which failed.

Reply:
Fully qualified domain names are actually [supposed to end with a dot](http://www.dns-sd.org/TrailingDotsInDomainNames.html), believe it or not -- so this is correct.

Reply:
Huh, I knew about this for domains (esp with regards to CNAMEs etc.), but hadn't realised it was applicable to email addresses, for whatever reason!